homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: done
+  appropriate-polling: done
+  entity-unique-id: done
+  has-entity-name: done
+  entity-event-setup: done
+  dependency-transparency: done
+  action-setup: done
+  common-modules: done
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  docs-actions: done
+  brands: done
+  ## Silver
+  config-entry-unloading: done
+  log-when-unavailable: done
+  entity-unavailable: done
+  action-exceptions: done
+  reauthentication-flow:
+    status: exempt
+    comment: This integration does not have reauthentication (e.g. fixed authentication for Plugwise Smiles)
+  parallel-updates:
+    status: done
+    comment: Using coordinator
+  test-coverage: done
```

Comment:
`test_sensor.py` line 161:

Instead of enabling entities on the fly
```py
    entity_registry.async_update_entity(entity_id=entity_id, disabled_by=None)
    await hass.async_block_till_done()

    await hass.config_entries.async_reload(init_integration.entry_id)
    await hass.async_block_till_done()
```
use a fixture:
```py
@pytest.mark.usefixtures("entity_registry_enabled_by_default")
```

Reply:
Yes, thanks, there are also some other similar test-improvements on its way...